scipy/linalg/flapack_other.pyf.src

```diff
@@ -1630,3 +1630,62 @@ subroutine ilaver(major, minor, patch)
     integer intent(out) :: minor
     integer intent(out) :: patch
 end subroutine ilaver
+
+
+
+subroutine <prefix2>gejsv(joba,jobu,jobv,jobr,jobt,jobp,m,n,a,lda,sva,u,ldu,v,ldv,work,lwork,iwork,info)
+    callstatement(*f2py_func)(joba,jobu,jobv,jobr,jobt,jobp,&m,&n,a,&lda,sva,u,&ldu,v,&ldv,work,&lwork,iwork,&info)
+    callprotoargument char*,char*,char*,char*,char*,char*,int*,int*,<ctype2>*,int*,<ctype2>*,<ctype2>*,int*,<ctype2>*,int*,<ctype2>*,int*,<ctype2>*,int*
+
+    character intent(in, optional), check(*joba=='C'||*joba=='E'||*joba=='F'||*joba=='G'||*joba=='A'||*joba=='R') :: joba='C'
+    character intent(in, optional), check(*jobu=='U'||*jobu=='F'||*jobu=='W'||*jobu=='N') :: jobu='U'
+    character intent(in, optional), check(*jobv=='V'||*jobv=='J'||*jobv=='W'||*jobv=='N') :: jobv='V'
+    character intent(in, optional), check(*jobr=='R'||*jobr=='N') :: jobr='R'
+    character intent(in, optional), check(*jobt=='N'||*jobt=='T') :: jobt='N'
+    character intent(in, optional), check(*jobp=='N'||*jobp=='P') :: jobp='N'
+
+    integer intent(hide),depend(a) ::m=shape(a,0)
+    integer intent(hide),depend(a) ::n=shape(a,1)
+    <ftype2> dimension(n,m), intent(in) :: a
```

Comment:
```suggestion
    <ftype2> dimension(m, n), intent(in) :: a
```
No?

Reply:
I tried this and I get a segfault. I'm confused. I thought it was row, col in these wrappers just like Python.

Reply:
Your suggestion is correct. Although that wasn't the issue. There were a few different reasons for the segfaults, but, I think that was because the real and complex values of ``lwork`` are slightly different another reason could be that u can be either m x  n or m x m (if jobu == 'F') although this is documented poorly for in the dgejsv documentation.